ui/app/AppLayouts/Chat/views/communities/CommunityNewCollectibleView.qml

```diff
@@ -178,20 +233,27 @@ StatusScrollView {
 
         CustomSwitchRowComponent {
             id: transferableChecker
-
+            visible: (optionsTab.currentIndex === 0)
```

Comment:
We can add a constant to identify option `collectible` or `asset` to make it more readable. However, I will branch from this branch and will start the development of #10623 from here, so no need to update that. Just focus on changes related to the image and drag/drop management!

Reply:
Done here: https://github.com/status-im/status-desktop/pull/10774